src/plugins/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx

```diff
@@ -153,6 +154,11 @@ export function InternalDashboardTopNav({
     title,
   ]);
 
+  /** Set chrome tab title when dashboard's title changes */
+  useEffect(() => {
+    chromeDocTitle.change(title);
```

Comment:
Can we fall back here to the default `New Dashboard` string here? Right now when you're on a new Dashboard the tab just says `Elastic`

Reply:
Good catch - done in 0383cb2624736cff2ef4eae69c1d90937106f6a7 :+1: